go/gen.py

```diff
@@ -2,8 +2,18 @@
 import pathlib
 import subprocess
 import os
+import argparse
+import shutil
 from python.runfiles import runfiles
 
+def options():
+    p = argparse.ArgumentParser(description="Update generated checked in files in the Go pack")
```

Comment:
Minor (to avoid confusion, since I'd view the "pack" as the release bundle):
```suggestion
    p = argparse.ArgumentParser(description="Update generated files related to Go in the repo")
```